python/tvm/contrib/tedd.py

```diff
@@ -78,6 +78,30 @@ def insert_dot_id(sch):
     return sch
 
 
+def itervar_equal(iv_a, iv_b):
+    """A helper method that compares the equality of two iterative variables"""
+    # after schedule.normalize was called, domains of iterative variables inside
+    # stage.leaf_iter_vars would be discarded and expected to be inferred in later
+    # InferBound call
+    # this rebase behavior would make plain comparison fail (i.e. ivar == itervar)
```

Comment:
What is "rebase behavior"

Reply:
Hi masahi,

Sorry for the unclear statement.
The rebase behavior here means the new bounds of  iterative variables, which is rebased from InferBound. Because original way (iter_var1 == inter_var2) would fail after invoking InferBound. We apply a new way to compare iterative variables.

Reply:
I mean, I'm not asking for an explanation for me. This is not a conventional use of the term "rebase". Please do not use a term that requires long explanation in doc or code.

Reply:
I see, thank you a lot for kindly point out the problem for us. :D

Really sorry for using a wrong term here. We remove the term of "rebase", and rephrase the comments a bit in the new patch.